pkg/server/server.go

```diff
@@ -37,53 +37,60 @@ func (server JSONRPCServer) commandHandler(_ context.Context, reply jsonrpc2.Rep
 		}
 	}()
 
+	logErrorAndReply := func(ctx context.Context, result interface{}, err error) error {
+		if err != nil {
+			rollbar.Log(rollbar.ERR, err)
```

Comment:
Is it possible to also print the error in the console ? So we can have feedback in the extension

Reply:
Great idea! I'll do just that.

Reply:
Done.